tools/dataset/pkg/dataset.go

```
@@ -0,0 +1,122 @@
+		return nil, err
+	}
+
+	cmp.RDS, err = NewRDSComponent(ctx, "rds", &RDSComponentArgs{
+		InstanceClass:     args.RDS.InstanceClass,
+		DBSubnetGroupName: args.RDS.DBSubnetGroupName,
+	}, pulumi.Parent(cmp))
+	if err != nil {
+		return nil, fmt.Errorf("creating RDS component: %w", err)
+	}
+
+	cmp.Ledger, err = pulumi_ledger.NewLedgerComponent(ctx, "ledger", &pulumi_ledger.LedgerComponentArgs{
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)